content/guides/autodiscovery.md

```diff
@@ -1,40 +1,75 @@
 ---
-title: Guide to using Autodiscovery with Docker
+title: Using Autodiscovery with Docker
 kind: guide
 listorder: 10
 ---
 
-Docker is being [adopted rapidly](https://www.datadoghq.com/docker-adoption/) and platforms like Docker Swarm, Kubernetes and Amazon's ECS make running services easier and more resilient by managing orchestration and replication across hosts. But all of that makes monitoring more difficult. How can you monitor a service which is dynamically shifting from one host to another?
+Docker is being [adopted rapidly](https://www.datadoghq.com/docker-adoption/). Orchestration platforms like Docker Swarm, Kubernetes, and Amazon ECS make running Docker-ized services easier and more resilient by managing orchestration and replication across hosts. But all of that makes monitoring more difficult. How can you reliably monitor a service which is unpredictably shifting from one host to another?
```

Comment:
is unpredictably the right word? We didn't like dynamically?

would "dynamically being shifted" make more sense?